app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(
+            self,
+            name: str,
+            age: int
+    ) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
```

Comment:
The use of the `people` dictionary as a class variable can lead to unexpected behaviour. Since it's shared among all instances of the `Person` class, changes to it will affect all instances, not just the one that made the change. Consider making `people` an instance variable instead, or refactor your code to avoid the need for it.